use-cases-working-draft.md

```diff
@@ -296,7 +296,7 @@ The requirements for Interchain NFTs include that these should be:
 5. **Non-transferable** to not permit changes in ownership.
 6. **Lockable** to conditionally stop changes in the token metadata.
 7. **Immutable** to never permit changes in the token metadata.
-8. **Mutable** to unconditionally permit changes in the token metadata.
+8. **Mutable** to conditionally permit changes in the token metadata.
```

Comment:
or should this be unconditionally? I'm confused why we need "Mutable" and "Conditional" at the same time. Isn't "Conditional" the same as "Mutable" without the "unconditionally"? When would a token need to "unconditionally permit changes"? Also are these changes meant to be restructed to Metadata or to ownership as well? Is Ownership just one possible item within metadata?

Reply:
Makes me wonder about "Ownable" as well, should it include the ability to change things like owneship? or is this confusion removed when it's noted that ownable is just one aspect of metadata but it's being highlighted as a special case?

Reply:
I think there are logically 3 requirements, which I can hopefully express more clearly with the following user stories:

As an NFT owner, I can:

1. Not make any changes to the token metadata after the token is minted, because this is immutable. 
2. Change the token metadata without any restrictions, which includes changing resource identifiers, because this is mutable.
3. Change the token metadata only under specified conditions (such as transfer in ownership), which may be limited to only specific metadata properties being changeable, because this is conditional.

Reply:
this makes sense to me